models/ios/acls/ios_acls.yaml

```diff
@@ -0,0 +1,679 @@
+---
+ANSIBLE_METADATA: |
+    {
+      'metadata_version': '1.1',
+      'status': ['preview'],
+      'supported_by': 'network'
+    }
+
+DOCUMENTATION: |
+  module: ios_acls
+  version_added: 2.10
+  short_description: Configure and manage Named or Numbered ACLs on IOS devices.
+  description: This module configures and manages the Named or Numbered on IOS platforms.
+  author: Sumit Jaiswal (@justjais)
+  notes:
+  - Tested against Cisco IOSv Version 15.2 on VIRL
+  - This module works with connection C(network_cli).
+    See L(IOS Platform Options,../network/user_guide/platform_ios.html).
+  options:
+    config:
+      description: A dictionary of ACL options
+      type: list
+      elements: dict
+      suboptions:
```

Comment:
There should be a top-level key indicating the address family to which the access list belongs. Otherwise, we wouldn't be able to differentiate between an IPv4 and IPv6 ACL.